docs/source/notebooks/Remote computing.ipynb

```diff
@@ -136,8 +136,9 @@
    "outputs": [],
    "source": [
     "# Use your key here to let the system know who you are\n",
-    "token_qcloud = 'YOUR_API_KEY'\n",
-    "remote_simulator = pcvl.RemoteProcessor(\"sim:ascella\", token_qcloud)"
+    "pcvl.save_token('YOUR_API_KEY')\n",
```

Comment:
Could you explain pcvl.save_token a bit more, in another cell to express the fact that it can be done once and for all.